MiniApp/Classes/Display/RealMiniAppView+WKUIDelegate.swift

```diff
@@ -0,0 +1,48 @@
+import WebKit
+
+extension RealMiniAppView: WKUIDelegate {
+    func webView(_ webView: WKWebView,
+                 runJavaScriptAlertPanelWithMessage message: String,
+                 initiatedByFrame frame: WKFrameInfo,
+                 completionHandler: @escaping () -> Void) {
+        let alertController = UIAlertController(title: "Info", message: message, preferredStyle: .alert)
+        alertController.addAction(UIAlertAction(title: "OK", style: .default, handler: nil))
+        UIApplication.topViewController()?.present(alertController, animated: true, completion: nil)
+        completionHandler()
+    }
+
+    func webView(_ webView: WKWebView,
+                 runJavaScriptConfirmPanelWithMessage message: String,
+                 initiatedByFrame frame: WKFrameInfo,
+                 completionHandler: @escaping (Bool) -> Void) {
+        let alertController = UIAlertController(title: "Info", message: message, preferredStyle: .alert)
+        alertController.addAction(UIAlertAction(title: "OK", style: .default, handler: { (_) in
+            completionHandler(true)
+        }))
+        alertController.addAction(UIAlertAction(title: "Cancel", style: .cancel, handler: { (_) in
```

Comment:
There's also the problem of localization. This text should be in Japanese on a device set to Japanese. But this could be a problem because maybe we would need to add support for other languages in the future. Does iOS have a built-in solution for this?

For example on Android, we can set string resources and then override them with locale-specific resource. Then if an App developer wants to add another language, they can override the string themselves by adding another local-specific resource for the language they want to support. Does iOS have something like this?

Reply:
Yes, we have Localization in iOS that will help to support different languages, will add it. 👍

Reply:
We already use localizable strings in the Demo app

Reply:
@Climbatize Yes, and this one is for SDK

Reply:
Done